### PR TITLE
feat(preset): add containerbase replacements

### PR DESCRIPTION
**Pull request by viceice**
## Changes
add replacements for `containerbase/buildpack` and `renovatebot/internal-tools`

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by secustor — DISMISSED** [review]

File: lib/config/presets/internal/replacements.ts [hunk 1]
```diff
@@ -104,6 +105,38 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  containerbase: {
+    description: 'replace containerbase stuff',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:109]:
```suggestion
    description: 'Replace containerbase dependencies.',
```

File: lib/config/presets/internal/replacements.ts [hunk 2]
```diff
@@ -104,6 +105,38 @@ export const presets: Record<string, Preset> = {
...
+    packageRules: [
+      {
+        description: 'replace containerbase/buildpack to containerbase/base',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:112]:
```suggestion
        description: 'Replace `containerbase/buildpack` with `containerbase/base`.',
```

File: lib/config/presets/internal/replacements.ts [hunk 3]
```diff
@@ -104,6 +105,38 @@ export const presets: Record<string, Preset> = {
...
+        matchDatasources: ['docker'],
+        matchPackageNames: ['containerbase/buildpack'],
+        replacementName: 'containerbase/base',
+      },
+      {
+        description:
+          'replace docker.io/containerbase/buildpack to docker.io/containerbase/base',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:119]:
```suggestion
          'Replace `docker.io/containerbase/buildpack` with `docker.io/containerbase/base`.',
```

File: lib/config/presets/internal/replacements.ts [hunk 4]
```diff
@@ -104,6 +105,38 @@ export const presets: Record<string, Preset> = {
...
+        matchDatasources: ['docker'],
+        matchPackageNames: ['docker.io/containerbase/buildpack'],
+        replacementName: 'docker.io/containerbase/base',
+      },
+      {
+        description:
+          'replace ghcr.io/containerbase/buildpack to ghcr.io/containerbase/base',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:126]:
```suggestion
          'Replace `ghcr.io/containerbase/buildpack` with `ghcr.io/containerbase/base`.',
```

File: lib/config/presets/internal/replacements.ts [hunk 5]
```diff
@@ -104,6 +105,38 @@ export const presets: Record<string, Preset> = {
...
+        matchDatasources: ['docker'],
+        matchPackageNames: ['ghcr.io/containerbase/buildpack'],
+        replacementName: 'ghcr.io/containerbase/base',
+      },
+      {
+        description:
+          'replace renovatebot/internal-tools to containerbase/internal-tools',
```
Comment (HonkingGoose) [inline lib/config/presets/internal/replacements.ts:133]:
```suggestion
          'Replace `renovatebot/internal-tools` with `containerbase/internal-tools`.',
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Fixing the grammar and styling of the descriptions.

**Review by rarkins — DISMISSED** [review]

**Review by secustor — APPROVED** [review]

**Pull request merged**